src/app/views/home_page.js

```diff
@@ -0,0 +1,39 @@
+import View from 'ampersand-view';
+import mustache from 'mustache';
+import ExperimentRowView from './experiment_row_view';
+
+export default View.extend({
+  template: mustache.render('<section class="page"><ul class="experiments"></ul></section>'),
```

Comment:
do we need mustache at all here? The docs I've seen look like template property can just be a string